lib/workers/repository/process/lookup/bucket.ts

```diff
@@ -19,6 +19,10 @@ export function getBucket(
   }
   const fromMajor = versioning.getMajor(currentVersion);
   const toMajor = versioning.getMajor(newVersion);
+  // istanbul ignore if
+  if (toMajor === null) {
```

Comment:
Maybe also check for undefined?

Reply:
Sorry, missed this. Will see if there's still any errors in the app ;)